### PR TITLE
Use `files` field instead of `.npmignore`

### PR DESCRIPTION
**Pull request by aaharu**
- [x] Maintain the existing code style
- [x] Are focused on a single change (i.e. avoid large refactoring or style adjustments in untouched code if not the primary goal of the pull request)
- [x] Have good commit messages
- [ ] Have tests
- [x] Don't significantly decrease the current code coverage (see coverage/lcov-report/index.html)

referenced #1407

These are removed files from package.
```diff
Only in node_modules/handlebars: .gitattributes
Only in node_modules/handlebars: .gitmodules
Only in node_modules/handlebars: .idea
Only in node_modules/handlebars: .istanbul.yml
Only in node_modules/handlebars: .npmignore
Only in node_modules/handlebars: CONTRIBUTING.md
Only in node_modules/handlebars: FAQ.md
Only in node_modules/handlebars: appveyor.yml
Only in node_modules/handlebars: docs
Only in node_modules/handlebars: package-lock.json
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,5 +73,15 @@
     "buildConfig": {
       "minify": true
     }
-  }
+  },
+  "files": [
+    "bin",
+    "dist/*.js",
```
Comment (nknapp) [inline package.json]: I think you could replace all the `dist/...`-entries with just one `dist` entry.

**Comment by nknapp** [comment]
I've added a small comment to the commit. I think one `dist` entry is sufficient. The rest looks good.
You don't need to open another PR for that change. Just push or amend and force-push. 
I'll squash-merge later if needed.

**Comment by nknapp** [comment]
I think I have forgot about the components... I'll have a look tonight, but I might just take your first commit after all.  Thanks.

**Comment by nknapp** [comment]
I have merged your first commit manually.

**Comment by nknapp** [comment]
Is it important that you have a release with this change anytime soon? Otherwise, I would wait for other changes.

**Comment by aaharu** [comment]
thank you.
It is not important for me 😸

**Pull request closed without merging**